internal/server/http/handler/v1/tree/handler.go

```diff
@@ -142,10 +142,18 @@ func UpdateTree(_ service.TreeService) fiber.Handler {
 // @Failure		500	{object}	HTTPError
 // @Router			/v1/tree [delete]
 // @Param			Authorization	header	string	true	"Insert your access token"	default(Bearer <Add access token here>)
```

Comment:
Please add the id to swagger

```suggestion
// @Router			/v1/tree/{tree_id} [delete]
// @Param			treecluster_id	query	string	false	"Tree Cluster ID"
// @Param			Authorization	header	string	true	"Insert your access token"	default(Bearer <Add access token here>)
```

Reply:
Why treecluster_id, you mean tree_id ?